doc/syntax.rst

```diff
@@ -318,8 +318,14 @@ Format specifications for chrono duration and time point types as well as
               : "q" | "Q" | "r" | "R" | "S" | "t" | "T" | "u" | "U" | "V" |
               : "w" | "W" | "x" | "X" | "y" | "Y" | "z" | "Z" | "%"
 
-Literal chars are copied unchanged to the output. Precision is valid only for
-``std::chrono::duration`` types with a floating-point representation type.
+Literal chars are copied unchanged to the output. For durations, the syntax is:
+
+.. productionlist:: sf
+   chrono_format_spec: [[`fill`]`align`][`width`]["." `precision`][`chrono_specs`]
+   chrono_specs: [`chrono_specs`] `literal_char`
```

Comment:
This is incorrect because durations support conversion specifiers. Let's correct it by removing this line which will also fix duplicate definition of `chrono_specs`.